src/main/java/com/strandls/user/service/impl/AuthenticationServiceImpl.java

```diff
@@ -81,6 +88,25 @@ public class AuthenticationServiceImpl implements AuthenticationService {
 	@Inject
 	private UserVerificationService verificationService;
 
+	public String generateSms(String otp, String type, User user) {
+		switch (type) {
+		case "forgotPassword":
```

Comment:
Enums are mostly used in cca-api because it demands high re-usability but most of the cases in other back end services we use simple strings in switch cases as it is not required in any other places .